packages/dev/core/src/Cameras/targetCamera.ts

```diff
@@ -133,7 +134,10 @@ export class TargetCamera extends Camera {
         }
 
         if (this.lockedTarget.absolutePosition) {
-            this.lockedTarget.computeWorldMatrix();
+            const lockedTarget = this.lockedTarget as AbstractMesh;
+            const m = lockedTarget.computeWorldMatrix();
+            // in some cases the absolute position resets externally, but doesn't update since the matrix is cached.
+            m.getTranslationToRef(lockedTarget.absolutePosition);

```

Comment:
It means some external code is updating `TransformNode.absolutePosition` with some wrong values, as we must copy back "by hand" the translation part of the matrix into it? If that is so, is it not a bug that should be fixed in this external code and not in the camera code?

Reply:
Not external code but the physics engine code. that would require making changes to the physics step and Cedric can't find the time fully debugging this right now.
This solution is adding 3 float assignments in reusable objects, so i find it to be a good solution without the need to make major changed to the physics step